src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,58 @@
+.wrapper {
+  background: var(--color-background-strong-default);
+  color: var(--color-text);
```

Comment:
@toiletgranny we don't have a `--color-text` token. Did you mean to use a different one?

Reply:
@bibixx Yeah, sorry, I'm a dumbass. I meant `var(--color-text-default)`. Fixed!